src/packages/sdk/src/__tests__/__snapshots__/engineCommands.test.ts.snap

```diff
@@ -27901,9 +27899,9 @@ Object {
 
 exports[`getDMMF big schema 1`] = `119480158`;
 
-exports[`getDMMF chinook introspected schema 1`] = `842601`;
+exports[`getDMMF chinook introspected schema 1`] = `842348`;
 
-exports[`getDMMF chinook introspected schema connectOrCreate 1`] = `842601`;
+exports[`getDMMF chinook introspected schema connectOrCreate 1`] = `842348`;
```

Comment:
What's this?

Reply:
The size/length of a JSON stringified dmmf...

Not super useful like this to know what changed...!

https://github.com/prisma/prisma/blob/2226db95f6ea7d47dc87a16e68b8de0e5dd2be9a/src/packages/sdk/src/__tests__/engineCommands.test.ts#L177-L199

Reply:
The length of the stringify dmmf :confused:

Reply:
You can give `toMatchSnapshot()` a string as single param, then it uses that to name the snapshot. Will do that in a new PR here then :p